### ISSUE TITLE
MaxRequestWorkers for Worker/Event Mode shows 400 and 16

### ISSUE THREAD

**Issue by richardforth** [richardforth/apache2buddy#250]
```
[ >> ] ServerLimit directive not found, assuming default values.
[ -- ] Your ServerLimit setting is 16.
[ >> ] ThreadsPerChild directive not found, assuming default values.
[ -- ] Your ThreadsPerChild setting is 25.
[ -- ] Your ThreadsPerChild setting for worker MPM is  25
[ -- ] Your ServerLimit setting for worker MPM is  16
[ >> ] MaxRequestWorkers directive not found, assuming default values.
[ -- ] Your MaxRequestWorkers setting is 400.
```

In report section...

...

```
Apache's MaxRequestWorkers directive:                         16       <--------- Current Setting
```

this should say 400  (16 childs x 25 threads)

**Comment by richardforth** [comment]
Starting at line 1882 (as of this writing)

```perl
		if ( ! $NOINFO ) { show_info_box();  print "Your MaxClients setting is ${CYAN}$maxclients${ENDC}.\n" }
	}

	# Check 16.01
	# Check if maxclients is more than ServerLimit
	# Then set maxclients to serverlimit if serverlimit is LESS than MaxClients
	our $maxclients;
	our $serverlimit;
	if ($maxclients > $serverlimit) {
		$maxclients = $serverlimit;
		 if ( ! $NOWARN ) { show_warn_box; print "MaxClients directive is higher than ServerLimit, using ServerLimit ($serverlimit) to apply calculations.\n" }
	}
```

**Comment by richardforth** [comment]
so check 16 set it to 400 (to account for Worker / event) 

but then 16.1 sets it back to 16 due to serverlimit.

Maybe set an exception to only execute that 16.1 code if we are in prefork?

**Comment by richardforth** [comment]
meh

**Comment by richardforth** [comment]
As I said in other issues related to worker mode, it is but a hack, since 99% of the code is geared towards prefork.

Worker mode has a concept of workers and threads per worker.

serverlimit will be 16 for the workers, and 25 will be threads per worker. I beleive we set maxclients abitrarily to 400 further back as there was no code to handle worker mode effectively (16 x 25 - 400).

The check at 16.1 was introduced to set realistic limits on maxclients based on serverlimit to show the more likely configuration given we know that apache limits maxclients to the serverlimit upper limit.

It seems that this broke how we displayed worker mode connections as we "made up" maxclients to be 400 as the sum of the defaults 16 and 25 respectively. So its right that serverlimit be set to 16 and it would still be 400 .

The dliema here is as we have no code to suppor worker properly, do I just ignore worker mode?


I dont really have the skill or time to spend writing code for worker mode, it was never in the original project either  (http://apachebuddy.pl) from which this was forked.
